server/browserstack/browserstack.js

```diff
@@ -0,0 +1,50 @@
+const browserstackLocal = require('browserstack-local');
+const browserstack = require('browserstack');
+const open = require('open');
+const testBrowsers = require('./test-browsers.json');
+const url = process.argv[2];
+const user = process.env.BROWSERSTACK_USERNAME;
+const key = process.env.BROWSERSTACK_KEY;
+const settings = {
+  "local": true,
+  "url": url,
+  "wait_time": "10",
+  "browsers": testBrowsers
+}
+const bs_local = new browserstackLocal.Local();
```

Comment:
On the client, the lint rules require camelCase vars. I know we're not linting here – should we be? cc. @jamesgorrie